release-to-candidate/action.yaml

```diff
@@ -69,12 +106,14 @@ runs:
     - name: Parse snap review information
       id: parse
       shell: bash
+      with:
+        yaml_path: ${{ steps.yaml-path.outputs.yaml_path }}
       run : |
         # Populate defaults
         echo "classic=false" >> "$GITHUB_OUTPUT"
 
         # Check for classic confinement and update the output if the snap is classic
-        if [[ "$(cat snap/snapcraft.yaml | yq -r '.confinement')" == "classic" ]]; then
+        if [[ "$(cat ${yaml_path} | yq -r '.confinement')" == "classic" ]]; then
```

Comment:
This same logic will need to be ported to the `test-snap-can-build` workflow

Reply:
Yes, working on all of them one by one